From 79f8e1b8cb2dca0d2ce56e4e7fa15d5c29b4d2e8 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Tue, 7 Jul 2026 23:18:03 -0400
Subject: [PATCH] webui: apply Traffic Manager rename + drop dead bwdpi tabs to
 menuTrees/menuTree.js (the file www/Makefile:916 ships for RT-BE96U; earlier
 a93094d386 edited modules/menuTree.js which the build overwrites)

---
 release/src/router/www/require/menuTrees/menuTree.js | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/release/src/router/www/require/menuTrees/menuTree.js b/release/src/router/www/require/menuTrees/menuTree.js
index f8f688dd9a..5bee44c74b 100644
--- a/release/src/router/www/require/menuTrees/menuTree.js
+++ b/release/src/router/www/require/menuTrees/menuTree.js
@@ -96,18 +96,14 @@ define(function(){
 				] 
 			},
 			{
-				menuName: "<#Adaptive_QoS#>",
+				menuName: "<#Menu_TrafficManager#>",
 				index: "menu_BandwidthMonitor", 
 				tab: [
-					{url: "AdaptiveQoS_Bandwidth_Monitor.asp", tabName: "<#Bandwidth_monitor#>"},
-					{url: "QoS_EZQoS.asp", tabName: "<#Adaptive_QoS#>"},
-					{url: "QoS_Stats.asp", tabName: "Classification"},
-					{url: "AdaptiveQoS_WebHistory.asp", tabName: "<#Adaptive_History#>"},
+					{url: "QoS_EZQoS.asp", tabName: "<#menu5_3_2#>"},
 					{url: "AdaptiveQoS_ROG.asp", tabName: "<table style='margin-top:-7px;'><tr><td><img src='/images/ROG_Logo.png' style='border:0px;width:32px;'></td><td>ROG First</td></tr></table>"}, 
 					{url: "Main_Spectrum_Content.asp", tabName: "<#Spectrum_title#>"},
 					{url: "Advanced_QOSUserPrio_Content.asp", tabName: "__INHERIT__"},
 					{url: "Advanced_QOSUserRules_Content.asp", tabName: "__INHERIT__"},
-					{url: "AdaptiveQoS_InternetSpeed.asp", tabName: "<#InternetSpeed#>"},
 					{url: "NULL", tabName: "__INHERIT__"}
 				] 
 			},		
-- 
2.25.1

